recipes/glew/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "glew" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "cc88d2f70a185185eb050eb13789f498425fcd58230bfd65da496b8008f046f2" %}
+{% set dev_url = "https://github.com/nigels-com/glew" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: {{ dev_url }}/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - CMakeLists.patch  # [win and not py35]
+  
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - cmake
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+# # Tests require visual context not present on CIs
```

Comment:
Agreeing with @scopatz that file tests are quite doable. Please take a look at [`boost`]( https://github.com/conda-forge/boost-feedstock/blob/dac7c7058a4c3b9f874ae11c89d0fcaeb1f41f0d/recipe/meta.yaml#L34-L36 ) for an example.